CONTRIBUTING_NI.md

```diff
@@ -71,3 +71,28 @@ git cherry-pick 095cea2^..d919979
 
 Carefully resolve any merge conflicts. Run `git cherry-pick --skip` for any changes that were accepted upstream since the last version bump.
 Once the cherry pick is completed, force push main (`git push -f`). If you are not an admin, ask one to do this step for you.
+
```

Comment:
@mollykreis also curious if this covers all the hiccups you found contributing to grafana as well